outaTime/outaTime/View Controllers/TimeCircuitsViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  TimeCircuitsViewController.swift
+//  outaTime
+//
+//  Created by Dojo on 6/9/20.
+//  Copyright © 2020 Dojo. All rights reserved.
+//
+
+import UIKit
+
+class TimeCircuitsViewController: UIViewController {
+    
+    @IBOutlet var destinationText: UILabel!
+    @IBOutlet var presentText: UILabel!
+    @IBOutlet var departedText: UILabel!
+    @IBOutlet var speedText: UILabel!
+    
+    @IBOutlet var travelBackButton: UIButton!
+    
+    @IBAction func travelBackTapped(_ sender: UIButton) {
+        startTimer()
+    }
```

Comment:
Maybe consider guarding against traveling back if the destination time is the same as the present time. You could for example disable the button until a new time is set, and then re-enable it.